Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Development Dockerfile for bioformats2raw
+# -----------------------------------------
+
+# To install the built distribution into other runtimes
+# pass a build argument, e.g.:
+#
+#   docker build --build-arg IMAGE=openjdk:9 ...
+#
+
+# Similarly, the BUILD_IMAGE argument can be overwritten
+# but this is generally not needed.
+ARG BUILD_IMAGE=gradle:5.2.1-jdk8
+
+#
+# Build phase: Use the gradle image for building.
+#
+FROM ${BUILD_IMAGE} as build
+USER root
+RUN apt-get update -qq && apt-get install -y -qq zeroc-ice-all-dev libblosc-dev
```

Comment:
Need zeroc-ice-all-dev?

Reply:
_Maybe_ not. I'll try to reduce the size, but it's certainly not optimized for size but for the dev-ness.